src/style.css

```
@@ -0,0 +1,87 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  margin-left: 48px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input-big {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  max-width: 900px;
+  height: 90px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  /* box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1); */
```

try without pixel for zero values

```suggestion
  /* box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1); */
```